tests/sentry/hybrid_cloud/test_log.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from sentry.models import AuditLogEntry, OutboxScope, RegionOutbox, UserIP
+from sentry.services.hybrid_cloud.log import AuditLogEvent, UserIpEvent, log_service
+from sentry.testutils.factories import Factories
+from sentry.testutils.silo import all_silo_test, exempt_from_silo_limits
+
+
+@pytest.mark.django_db
+@all_silo_test
+def test_audit_log_event():
+    user = Factories.create_user()
+    organization = Factories.create_organization()
+    log_service.record_audit_log(
+        event=AuditLogEvent(
+            organization_id=organization.id,
+            actor_user_id=user.id,
+            event_id=1,
+            ip_address="127.0.0.1",
+            target_object_id=123,
+            data=dict(abcdef=123),
+        )
+    )
+
+    with exempt_from_silo_limits():
```

Comment:
Rather than using exempt here, does it make more sense to have a test utility specifically for draining the outbox that encapsulated the exempt? e.g.
```
with_drained_outbox():
    ...blah...
assert stuff_happened
```